### ISSUE TITLE
Splunk apps_location from default.yml?

### ISSUE THREAD

**Issue by aegershman** [splunk/docker-splunk#276]
Using `splunk/splunk:8.0.0`

If passing in `splunk.apps_location[]` values to the ansible config as code (described in [`defaults.yml`](https://github.com/splunk/splunk-ansible/blob/develop/docs/ADVANCED.md#apps)), none of the URLs appear to be registered. When dumping out the ansible vars during provisioning, it shows an empty array `[]`. Only when env vars for `SPLUNK_APPS_URL` are passed do any apps get registered.

[Are apps only registered in this container when `SPLUNK_APPS_URL` are passed via `env` and values to the ansible config ignored/overridden?](https://github.com/splunk/splunk-ansible/blob/develop/inventory/environ.py#L172-L176) 

Just checking, can provide more details if interested. Feel free to close whenever.

**Comment by mikedickey** [comment]
I noticed the same issue, and believe it's a bug in splunk-ansible. In particular this code appears to clobber any values passed via defaults.yml:

```
    apps = os.environ.get("SPLUNK_APPS_URL", None)
    if apps:
        vars_scope["splunk"]["apps_location"] = apps.split(",")
    else:
        vars_scope["splunk"]["apps_location"] = []
```

We have an internal issue CSPL-160 also tracking this.

**Comment by nwang92** [comment]
Closing because the PR was merged - this should be available in versions >=7.3.3 (ETA 11/19) and >=8.0.1. I'll also add a test for this.

**Comment by mikedickey** [comment]
I just fixed another bug related to this in splunk-ansible: https://github.com/splunk/splunk-ansible/pull/312/files
